Dockerfile-test_browser

```diff
@@ -44,15 +44,6 @@ COPY test test
 COPY webpack webpack
 COPY templates templates
 
-ENV NODE_ENV "production"
-ARG CI_COMMIT_SHA
```

Comment:
this is being removed, but it's not in `Dockerfile-app`

Reply:
`Dockerfile-app` does specify `ENV NODE_ENV "production"` ... but we no longer use the `CI_COMMIT_SHA` in the served pages.